From: PupJune2 <PupJune2@users.noreply.github.com>
Date: Fri, 16 Oct 2026 14:00:00 +0000
Subject: CromIKO App Name

Show the browser as CromIKO:
- launcher label, app_name and the home screen widget titles
- PRODUCT_FULLNAME/PRODUCT_SHORTNAME (about and version pages)
- the build-time string rewrite in grit now turns Chrome, Chromium,
  Bromite and Cromite into CromIKO; the "Cromite.org" credit and
  lowercase links to the Cromite repository and filter lists are kept
- Cromite-specific labels that are not in grd files (flags page, site
  settings category, flags shortcut)

The package name, Windows profile paths and technical identifiers are
unchanged.

License: GPL-3.0-only - https://spdx.org/licenses/GPL-3.0-only.html
---
 chrome/android/java/AndroidManifest.xml                   | 2 +-
 chrome/android/java/res/values/values.xml                 | 2 +-
 .../java/res_chromium_base/values/channel_constants.xml   | 8 ++++----
 chrome/app/theme/chromium/BRANDING                        | 8 ++++----
 .../android/java/res/xml/site_settings_preferences.xml    | 2 +-
 components/webui/flags/resources/app.ts                   | 4 ++--
 components/webui/flags/resources/experiment.ts            | 2 +-
 tools/grit/grit/grd_reader.py                             | 7 +++++++
 8 files changed, 21 insertions(+), 14 deletions(-)

diff --git a/chrome/android/java/AndroidManifest.xml b/chrome/android/java/AndroidManifest.xml
--- a/chrome/android/java/AndroidManifest.xml
+++ b/chrome/android/java/AndroidManifest.xml
@@ -199,7 +199,7 @@ by a child template that "extends" this file.
         android:gwpAsanMode="always"
         android:icon="@drawable/ic_launcher"
         android:roundIcon="@drawable/ic_launcher_round"
-        android:label="Cromite"
+        android:label="CromIKO"
         android:memtagMode="async"
         android:largeHeap="false"
         android:manageSpaceActivity="@string/manage_space_activity"
diff --git a/chrome/android/java/res/values/values.xml b/chrome/android/java/res/values/values.xml
--- a/chrome/android/java/res/values/values.xml
+++ b/chrome/android/java/res/values/values.xml
@@ -9,7 +9,7 @@ found in the LICENSE file.
     xmlns:tools="http://schemas.android.com/tools"
     tools:ignore="MissingTranslation">
 
-    <string name="cromite_flags_title">Open Cromite flags list</string>
+    <string name="cromite_flags_title">Open CromIKO flags list</string>
     <string name="cromite_flags_url">chrome://flags/cromite</string>
 
     <!-- Full Screen Constants -->
diff --git a/chrome/android/java/res_chromium_base/values/channel_constants.xml b/chrome/android/java/res_chromium_base/values/channel_constants.xml
--- a/chrome/android/java/res_chromium_base/values/channel_constants.xml
+++ b/chrome/android/java/res_chromium_base/values/channel_constants.xml
@@ -7,8 +7,8 @@ found in the LICENSE file.
 
 <resources xmlns:android="http://schemas.android.com/apk/res/android">
     <!-- The application name displayed to the user. -->
-    <string name="app_name" translatable="false">Cromite</string>
-    <string name="bookmark_widget_title" translatable="false">Cromite bookmarks</string>
-    <string name="search_widget_title" translatable="false">Cromite search</string>
-    <string name="quick_action_search_widget_title" translatable="false">Cromite quick action search</string>
+    <string name="app_name" translatable="false">CromIKO</string>
+    <string name="bookmark_widget_title" translatable="false">CromIKO bookmarks</string>
+    <string name="search_widget_title" translatable="false">CromIKO search</string>
+    <string name="quick_action_search_widget_title" translatable="false">CromIKO quick action search</string>
 </resources>
diff --git a/chrome/app/theme/chromium/BRANDING b/chrome/app/theme/chromium/BRANDING
--- a/chrome/app/theme/chromium/BRANDING
+++ b/chrome/app/theme/chromium/BRANDING
@@ -1,9 +1,9 @@
 COMPANY_FULLNAME=The Chromium Authors
 COMPANY_SHORTNAME=The Chromium Authors
-PRODUCT_FULLNAME=Cromite
-PRODUCT_SHORTNAME=Cromite
-PRODUCT_INSTALLER_FULLNAME=Cromite Installer
-PRODUCT_INSTALLER_SHORTNAME=Cromite Installer
+PRODUCT_FULLNAME=CromIKO
+PRODUCT_SHORTNAME=CromIKO
+PRODUCT_INSTALLER_FULLNAME=CromIKO Installer
+PRODUCT_INSTALLER_SHORTNAME=CromIKO Installer
 COPYRIGHT=Copyright @LASTCHANGE_YEAR@ The Chromium Authors. All rights reserved.
 MAC_BUNDLE_ID=org.chromium.Chromium
 MAC_CREATOR_CODE=Cr24
diff --git a/components/browser_ui/site_settings/android/java/res/xml/site_settings_preferences.xml b/components/browser_ui/site_settings/android/java/res/xml/site_settings_preferences.xml
--- a/components/browser_ui/site_settings/android/java/res/xml/site_settings_preferences.xml
+++ b/components/browser_ui/site_settings/android/java/res/xml/site_settings_preferences.xml
@@ -198,6 +198,6 @@ The order of the following items is from: http://crbug.com/610358.
         android:selectable="false"
         android:layout="@layout/horizontal_divider"/>
 
-    <PreferenceCategory android:title="Cromite content settings"/>
+    <PreferenceCategory android:title="CromIKO content settings"/>
 
 </PreferenceScreen>
diff --git a/components/webui/flags/resources/app.ts b/components/webui/flags/resources/app.ts
--- a/components/webui/flags/resources/app.ts
+++ b/components/webui/flags/resources/app.ts
@@ -176,7 +176,7 @@ export class FlagsAppElement extends CrLitElement {
     // <if expr="not is_ios">
     loadTimeData.getString('unavailable'),
     // </if>
-    "Cromite",
+    "CromIKO",
   ];
   protected accessor selectedTabIndex_: number = 0;
 
@@ -238,7 +238,7 @@ export class FlagsAppElement extends CrLitElement {
     if (location.pathname == '/cromite') {
       this.onlyCromiteFlags = true;
       this.getRequiredElement("#appcontainer").classList.add('cromite');
-      document.title = "Cromite Flags List";
+      document.title = "CromIKO Flags List";
     }
     // <if expr="not is_ios">
     const pathname = new URL(window.location.href).pathname;
diff --git a/components/webui/flags/resources/experiment.ts b/components/webui/flags/resources/experiment.ts
--- a/components/webui/flags/resources/experiment.ts
+++ b/components/webui/flags/resources/experiment.ts
@@ -132,7 +132,7 @@ export class ExperimentElement extends CrLitElement {
   protected getExperimentTitle_(): string {
     const suffix =
       this.feature_.is_cromite && this.feature_.is_new
-        ? " (Cromite flag)" : "";
+        ? " (CromIKO flag)" : "";
     if (this.showEnableDisableSelect_()) {
       return (this.isDefault_ ? '' :
                                loadTimeData.getString('experiment-enabled')) + suffix;
diff --git a/tools/grit/grit/grd_reader.py b/tools/grit/grit/grd_reader.py
--- a/tools/grit/grit/grd_reader.py
+++ b/tools/grit/grit/grd_reader.py
@@ -177,6 +177,13 @@ class GrdContentHandler(xml.sax.handler.ContentHandler):
     content = content.replace("zzzyyyyxxxx8", ".chromium.org")
     content = content.replace("zzzyyyyxxxx9", "chromebook")
     content = content.replace("zzzyyyyxxxxA", "chromeos")
+
+    # CromIKO: show the product as CromIKO, but keep the Cromite.org credit.
+    # Lowercase "cromite" is left alone so that links to the Cromite repository
+    # and its filter lists keep working.
+    content = content.replace("Cromite.org", "zzzyyyyxxxxB")
+    content = content.replace("Cromite", "CromIKO")
+    content = content.replace("zzzyyyyxxxxB", "Cromite.org")
     # if "dontreplace" in content_orig:
     #   raise exception.MissingElement("stop")
     #   print(content + " <-- " + content_orig)
--
